subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -43,6 +43,14 @@ config BT_EATT_MAX
 	help
 	  Number of Enhanced ATT bearers available.
 
+
+config BT_EATT_AUTO_CONNECT
+	bool "Automatically connect EATT bearers when a link is established"
+	default y
```

Comment:
Default on or off?

Reply:
It makes sense for me to automatically connect.
But it may be an idea to only have the initiator/central automatically connect for IOP reasons.
It is not a requirement for the spec, but it is more in line with the role of the initiator. It will also avoid a lot of collisions.